TestAssets/TestPackages/dotnet-portable/Program.cs

```diff
@@ -10,6 +12,12 @@ public class Program
         public static void Main(string[] args)
         {
             Console.WriteLine("Hello Portable World!");
+
+            var coreAssembly = typeof(object).GetTypeInfo().Assembly;
+            string coreFolder = Path.GetDirectoryName(coreAssembly.Location);
+            string frameworkVersion = Path.GetFileName(coreFolder);
```

Comment:
The more correct way to do this is to do it the way the Muxer does it:  [https://github.com/dotnet/cli/blob/master/src/Microsoft.DotNet.Cli.Utils/Muxer.cs#L21-L22](https://github.com/dotnet/cli/blob/master/src/Microsoft.DotNet.Cli.Utils/Muxer.cs#L21-L22)

use the `FX_DEPS_FILE` passed through the appcontext.  The reasoning is that servicing can make it so the framework assemblies are loaded from a "servicing" folder, and not from the shared fx folder.